### PR TITLE
bug Fix - Modify conversion string to number

### PR DESCRIPTION
**Pull request by patapufate**
Modify the conversion of the message sent by Diagnostics.

### CODE REVIEW

File: health_monitor/src/health_monitor.cpp [hunk 1]
```diff
@@ -108,7 +108,7 @@ void HealthMonitor::handle_diagnostics(const diagnostic_msgs::DiagnosticArrayPtr
             (msg->status[i].values.size() > 0) &&
             (msg->status[i].values[0].key == "Code"))
         {
-            errorValues |= stoi(msg->status[i].values[0].value);
+            errorValues |= stol(msg->status[i].values[0].value);
```
Comment (hidmic) [inline health_monitor/src/health_monitor.cpp:111]: @patapufate just to be on the safe side, use `stoll` instead.

**Review by hidmic — CHANGES_REQUESTED** [review]

File: health_monitor/src/health_monitor.cpp (hunk 1 above)
Comment (ChrisScianna) [reply health_monitor/src/health_monitor.cpp:111]: I was going to say the same. I tested with stoll. If you'd prefer I can test again with stol

Comment (patapufate) [reply health_monitor/src/health_monitor.cpp:111]: As you and @hidmic have said, I think it will be better if we use stoll instead.

**Review by hidmic — APPROVED** [review]
LGTM !

**Pull request merged**